pkgs/development/python-modules/pyqt/5.x.nix

```diff
@@ -32,9 +32,9 @@ in buildPythonPackage {
 
   outputs = [ "out" "dev" ];
 
-  nativeBuildInputs = [ pkgconfig qmake lndir ];
+  nativeBuildInputs = [ pkgconfig qmake lndir sip ];
```

Comment:
Should this be pyqt5-sip?

Reply:
No, it seems that the upstream envisions that there is one tool `sip` which generates multiple modules according to `sip -n module_name` parameter. This is why `sip`-the-tool is a native build input, and `pyqt5-sip`-module is a propagated build input.

After this PR `sip` ships both the tool and the module `sip` (as before), but `pyqt5-sip` ships only the module `PyQt5.sip`.

Reply:
Ok. Could you include the explanation as a comment?